src/qtt/algorithms/functions.py

```diff
@@ -27,6 +27,20 @@ def gaussian(x, mean, std, amplitude=1, offset=0):
     y = offset + amplitude * np.exp(- (x - mean) * (x - mean) / (2 * std * std))
     return y
 
+def sine(x, amplitude, frequency, phase, offset):
```

Comment:
Do type annotation
```suggestion
def sine(x : Union[float, np.ndarray], amplitude : float, frequency : float, phase : float, offset : float): Union[float, np.ndarray]
```
@QFer Agree to start adding type annotations?

Reply:
@peendebak I think it is wise to start with type annotations now more people are using (and maintaining) the code. It helps us and other users to see what types are used. At a later moment we can start typing whole files but for now do it for new stuff only (or when boyscouting). Is that oke with you Pieter?

Reply:
Check your pep messages. There should be 2 newlines between local methods. So an extra line before and after the sine function.

Reply:
Suggestion (maybe think over it or we can do it later): Isn't frequency = 2*pi / period? Maybe also add a parameter to give period. So we can call with period or the frequency. Before return we should then check if only one of them is given and not both because to prevent inconsistency.

Reply:
@QFer @bogdanvconstantin Good to note!

Currently we have sin(omega*t), where omega is the angular frequency. It would be better to use sin(2*pi*w*t), where w is the ordinary frequency. This is also more consistent with the current definition of `gauss_ramsey`. 
Bogdan: can you update to sin(2*pi*frequency*t)?

Reply:
adding a 2 * np.pi in the formula does something odd to the Model and the fit parameters are nowhere near the test parameters. I will not add this and we will have to extract it from omega if we need.